docs/content/ecs/logging.md

```diff
@@ -13,7 +13,7 @@ All workers use the `ForwardingDispatcher` by default. To replace it with the `L
 
```

Comment:
Re: line 10: Can we drive-by refine this line? Feels weird to read and I'm not sure what "forwards it" is referring to.